packages/core/src/plugins/web3Form.ts

```diff
@@ -0,0 +1,150 @@
+import { DomElement } from 'htmlparser2';
+import cheerio from 'cheerio';
+import has from 'lodash/has';
+import { PluginContext } from './Plugin';
+
+const md = require('../lib/markdown-it');
+
+const _ = {
+  has,
+};
+
+const CSS_FILE_NAME = 'web3FormAssets/web-3-form.css';
+
+const NAME_INPUT = '<label for="name">Name</label>\n'
+    + '<input type="text" name="name" required placeholder="John Doe">';
+const EMAIL_INPUT = '<label for="email">Email</label>\n'
+    + '<input type="email" name="email" required placeholder="johndoe@gmail.com">';
+const MESSAGE_INPUT = '<label for="message">Message</label>\n'
+    + '<textarea name="message" required></textarea>';
+const SUBMIT_BUTTON = '<button type="submit" class="badge bg-info">Submit</button>';
+
+const DEFAULT_INPUTS = `${NAME_INPUT}\n${EMAIL_INPUT}\n${MESSAGE_INPUT}\n${SUBMIT_BUTTON}`;
+
+function createMinimalForm(pluginContext: PluginContext) {
+  const $replaceNode = cheerio('<form onsubmit="submitForm(this)"></form>');
+  $replaceNode.append(`<input type="hidden" name="access_key" value=${pluginContext.accessKey}>`);
+  return $replaceNode;
+}
+
+function deleteWeb3FormAttributes(node: cheerio.Element & DomElement) {
+  const nodeAttribs = node.attribs;
+  if (!nodeAttribs) return;
+  delete nodeAttribs.default;
+  delete nodeAttribs.header;
+}
+
+function transformFormInputs(child: cheerio.Element) {
+  if (child.type !== 'tag') {
+    return;
+  }
+  const $node = cheerio(child);
+  switch (child.name.toLowerCase()) {
+  case 'name-input':
+    $node.replaceWith(NAME_INPUT);
+    break;
+  case 'email-input':
+    $node.replaceWith(EMAIL_INPUT);
+    break;
+  case 'message-input':
+    $node.replaceWith(MESSAGE_INPUT);
+    break;
+  case 'submit-button':
+    $node.replaceWith(SUBMIT_BUTTON);
+    break;
+  default:
+  }
+}
+
+function generateFormContainer(node: cheerio.Element & DomElement) {
+  if (node.type !== 'tag') {
+    return;
+  }
+  node.name = 'box';
+  const $node = cheerio(node);
+  if (!_.has(node.attribs, 'type')) {
+    $node.attr('type', 'info');
+  }
+  $node.attr('no-icon', '');
+  let header = '<h2>Contact Us</h2>';
```

Comment:
Hmm I'm not sure about having a (undocumented) default header here. On one hand it makes sense since this form is mostly used for "Contact Us" purposes; on the other hand, it's slightly weird to have to put `header=""` if you don't want a header. 
Either direction sounds good to me, wdyt?

Reply:
Oh that's a good point. I think it would make more sense to remove the header in that case. Thank you!

Reply:
I think it would be good to have it only if 1. its a default contact form and users do not specify a header. If they do, then the default header will be overriden